src/components/assets/Assets.vue

```diff
@@ -28,14 +29,33 @@ export default defineComponent({
   },
   setup() {
     const { tokens, ttlErc20Amount } = useCbridgeV2();
+    const isDisplay = ref<boolean>(false);
     const store = useStore();
     const selectedAddress = computed(() => store.getters['general/selectedAddress']);
     const isH160 = computed(() => store.getters['general/isH160Formatted']);
+
+    const setIsDisplay = (): void => {
+      const address = localStorage.getItem(LOCAL_STORAGE.SELECTED_ADDRESS);
+      const isEthereumExtension = address === 'Ethereum Extension';
+      if (!isDisplay.value && isEthereumExtension) {
+        setTimeout(() => {
+          isDisplay.value = true;
+        }, 1000);
```

Comment:
any specific reason using setTimeout for this? if so, 1000 should be set as constant value. others need to be aware that it takes this much time for some reason.

Reply:
@sirius651 
Without setTimeout (I know this is not an elegant solution), we will see the empty 'Native Account' component whenever users select EVM wallet. Sure, I've set the variable [here](https://github.com/AstarNetwork/astar-apps/pull/327/commits/d1251f298c1a88aacc4d3a34ddb6bf53a03b2003).

Demo
https://www.loom.com/share/02c0ffa61ac943ccba7bbd3666db19de

![image](https://user-images.githubusercontent.com/92044428/161672996-6c2c613d-c7d8-46ce-998a-c826b5afc92b.png)